build.xml

```diff
@@ -7,8 +7,8 @@
 <!-- the Compile on Save feature is turned off for the project. -->
 <!-- You can turn off the Compile on Save (or Deploy on Save) setting -->
 <!-- in the project's Project Properties dialog box.-->
-<project name="CharacterCounter" default="default" basedir=".">
-    <description>Builds, tests, and runs the project CharacterCounter.</description>
+<project name="CharacterCount" default="default" basedir=".">

```

Comment:
I'm not sure what you guys are doing and I know that I help you fix this yesterday, but why is the project name changing?